docs/UserGuide.md

```diff
@@ -425,16 +425,15 @@ This will result in the removal of the applicant from the pinned applicant list.
 
 Format: 
 ```
-unpin NAME
+unpin FULL_NAME
 ```
 
-* Unpins the scholarship applicant with the specified `NAME`.
-* `NAME` refers to the name of the scholarship applicant.
-* `NAME` **must be a full name:** `Bernice Yu` instead of `Bernice`.
-* Unpin Command is **case-insensitive**. e.g. `BernICE yU` will match the applicant with `NAME` `Bernice Yu`.
+* Unpins the scholarship applicant with the specified `FULL_NAME`.
+* `FULL_NAME` **must be a full name:** `Bernice Yu` instead of `Bernice`.
```

Comment:
maybe change this to something like
`FULL_NAME` must be the full name of an applicant. e.g. `Bernice Yu` instead of `Bernice`

Reply:
ok